pkg/printer/printer.go

```diff
@@ -272,11 +272,18 @@ func (p *Printer) WriteProtoFlow(res *observerpb.GetFlowsResponse) error {
 		if p.opts.nodeName {
 			node = fmt.Sprintf(" [%s]", f.GetNodeName())
 		}
+		arrow := "->"
+		if f.GetIsReply() != nil && f.GetIsReply().Value {
```

Comment:
Random thought: Could we set use a third arrow style in case `GetIsReply` is `nil`? i.e. when don't know if it is a reply or not? e.g. `<->` or something.